assignments/closure.js

```diff
@@ -1,21 +1,40 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
+const food = 'peach';
+function fruitSalad () {
```

Comment:
You're on the right track, but if you want fruitSalad to be a closure, it and food should be inside a parent function. A closure essentially means that a function has access to what's inside its parent function.